suites/fs/recovery/tasks/mds-full.yaml

```diff
@@ -5,6 +5,10 @@ overrides:
             osd:
               osd objectstore: memstore
               memstore device bytes: 100000000
+overrides:
+  ceph:
+    log-whitelist:
+      - OSD full dropping all updates
```

Comment:
We can't duplicate yaml sections like this; you need to merge the log-whitelist into the previous Ceph one. :)